### PR TITLE
Implement new intake functions

### PR DESCRIPTION
**Pull request by SUPERCILEX**

### CODE REVIEW

**Comment by SUPERCILEX** [comment]
@evanmyersflow `top` _is_ button `2` 😄.

File: src/main/java/org/sert2521/powerup/drivetrain/Drivetrain.kt [hunk 1]
```diff
@@ -40,7 +45,10 @@ object Drivetrain : Subsystem() {
         rightDrive.setSelectedSensor(FeedbackDevice.QuadEncoder)
     }
 
-    override fun onStart() = reset()
+    override fun onStart() {
+        reset()
+        rightJoystick.whenActive(12, SendToBottom() and DriveToCube())
+    }
```
Comment (andrewda) [inline src/main/java/org/sert2521/powerup/drivetrain/Drivetrain.kt]: not in this pr

File: src/main/java/org/sert2521/powerup/drivetrain/commands/AngleDriver.kt [hunk 2]
```diff
@@ -4,6 +4,10 @@ import org.sert2521.powerup.drivetrain.Drivetrain
 import org.sertain.command.PidCommand
 
 abstract class AngleDriver(p: Double, i: Double = 0.0, d: Double = 0.0) : PidCommand(p, i, d) {
+    init {
+        requires(Drivetrain)
+    }
```
Comment (andrewda) [inline src/main/java/org/sert2521/powerup/drivetrain/commands/AngleDriver.kt]: not in this pr

File: src/main/java/org/sert2521/powerup/drivetrain/commands/DriveToCube.kt [hunk 3]
```diff
@@ -7,7 +7,7 @@ import org.sert2521.powerup.elevator.Elevator
 import org.sert2521.powerup.intake.Intake
 import org.sert2521.powerup.util.DEGREES_PER_PIXEL
 
-class DriveToCube : AngleDriver(0.01) {
+class DriveToCube : AngleDriver(0.01, 0.0, 0.022) {
```
Comment (andrewda) [inline src/main/java/org/sert2521/powerup/drivetrain/commands/DriveToCube.kt]: not in this pr

File: src/main/java/org/sert2521/powerup/drivetrain/commands/TurnToAngle.kt [hunk 4]
```diff
@@ -6,10 +6,6 @@ import kotlin.math.absoluteValue
 class TurnToAngle(private val angle: Double) : AngleDriver(1.0) {
     private val startAngle by lazy { Drivetrain.ahrs.yaw }
 
-    init {
-        requires(Drivetrain)
-    }
-
```
Comment (andrewda) [inline src/main/java/org/sert2521/powerup/drivetrain/commands/TurnToAngle.kt]: not in this pr

**Pull request merged**